### PR TITLE
Adding VerifyBiosAttributes functionality

### PR DESCRIPTION
**Pull request by TSKushal**
SUMMARY
Adding functionality to verify user provided BIOS attributes with those on the server.

ISSUE TYPE
- Feature Pull Request

COMPONENT NAME
redfish_command.py
redfish_utils.py

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -1036,7 +1036,7 @@ files:
   $modules/read_csv.py:
     maintainers: dagwieers
   $modules/redfish_:
-    ignore: jose-delarosa
+    ignore: jose-delarosa varini-hp prativa-n Gayathirideviramasamy TSKushal
```
Comment (mraineri) [inline .github/BOTMETA.yml:1039]: Why is this needed?

File: plugins/modules/redfish_command.py [hunk 2]
```diff
@@ -239,8 +239,19 @@
     type: bool
     default: false
     version_added: 3.7.0
+  bios_attributes:
+    required: false
+    description:
+      - BIOS attributes that needs to be verified in the given server
+    type: dict
+    version_added: 6.1.0
```
Comment (felixfontein) [inline plugins/modules/redfish_command.py:247]:
```suggestion
    version_added: 6.3.0
```
(Since the 6.3.0 release is on Tuesday, 6.4.0 is probably more likely.)

File: plugins/modules/redfish_command.py [hunk 3]
```diff
@@ -239,8 +239,19 @@
     type: bool
     default: false
     version_added: 3.7.0
+  bios_attributes:
+    required: false
+    description:
+      - BIOS attributes that needs to be verified in the given server
```
Comment (felixfontein) [inline plugins/modules/redfish_command.py:245]:
```suggestion
      - BIOS attributes that needs to be verified in the given server.
```

File: plugins/modules/redfish_command.py [hunk 4]
```diff
@@ -239,8 +239,19 @@
...
+    type: dict
+    version_added: 6.1.0
 
-author: "Jose Delarosa (@jose-delarosa)"
+author:
+  - "Jose Delarosa (@jose-delarosa)"
+  - "Gayathiri Devi Ramasamy (@Gayathirideviramasamy)"
+  - "T S Kushal (@TSKushal)"
+  - "Varni H P (@varini-hp)"
+  - "Prativa Nayak (@prativa-n)"
```
Comment (felixfontein) [inline plugins/modules/redfish_command.py:254]: Same question here as in #5899.

File: .github/BOTMETA.yml (hunk 1 above)
Comment (TSKushal) [reply .github/BOTMETA.yml:1039]: Hi, 
This was raising an error in one if the tests when we tried to do it without adding this before.

File: plugins/modules/redfish_command.py (hunk 2 above)
Comment (TSKushal) [reply plugins/modules/redfish_command.py]: Changed to 6.4.0. Thanks!

File: .github/BOTMETA.yml [hunk 5]
```diff
@@ -1036,7 +1036,7 @@ files:
   $modules/read_csv.py:
     maintainers: dagwieers
   $modules/redfish_:
-    ignore: jose-delarosa
+    ignore: jose-delarosa TSKushal
```
Comment (mraineri) [inline .github/BOTMETA.yml:1039]: I'm still not sure why this is needed; I'm not entirely sure what it's supposed to control with the CI workflow. @felixfontein, can you explain what the ignore statement does here? I wasn't able to find much in the bot's documentation.

Comment (felixfontein) [reply .github/BOTMETA.yml:1039]: Every person mentioned in the `author` field of a module (or plugin) needs to show up in BOTMETA.yml as well, either in `maintainers` or in `ignore`. Usually adding yourself on the `authors` list means that you should at least for some time also serve as a maintainer for that module, so directly adding yourself to `ignore` is not a good idea.

Comment (felixfontein) [reply .github/BOTMETA.yml:1039]: About what `ignore` does: it basically says "do not ping this person when there is a PR/issue that affects this module".

Comment (mraineri) [reply .github/BOTMETA.yml:1039]: Got it; that makes sense why Jose is there since he's the original author, but has moved on from that job.

Comment (TSKushal) [reply .github/BOTMETA.yml:1039]: Hope this is clarified now. Can I resolve it?

File: plugins/modules/redfish_command.py (hunk 4 above)
Comment (TSKushal) [reply plugins/modules/redfish_command.py]: Resolving this conversation, as we discussed and closed this in PR [5899](https://github.com/ansible-collections/community.general/pull/5899). Thanks!

File: .github/BOTMETA.yml (hunk 1 above)
Comment (TSKushal) [reply .github/BOTMETA.yml]: Hope this is clarified now. Can I resolve it?

File: .github/BOTMETA.yml (hunk 5 above)
Comment (mraineri) [reply .github/BOTMETA.yml:1039]: @TSKushal given that you're adding yourself as an author, you should instead add yourself as a maintainer instead of adding yourself to ignore immediately.

Comment (TSKushal) [reply .github/BOTMETA.yml]: Done. Looking forward to working with everyone 😄

**Comment by mraineri** [comment]
shipit

**Comment by felixfontein** [comment]
Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by TSKushal** [comment]
> Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

Done 😄

File: changelogs/fragments/5900-adding-verifybiosattribute-fucntionality-to-redfish-command.yml [hunk 6]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_command module - adding VerifyBiosAttributes functionality (https://github.com/ansible-collections/community.general/pull/5900).
```
Comment (felixfontein) [inline changelogs/fragments/5900-adding-verifybiosattribute-fucntionality-to-redfish-command.yml:2]:
```suggestion
  - redfish_command - adding ``VerifyBiosAttributes`` functionality (https://github.com/ansible-collections/community.general/pull/5900).
```

**Comment by felixfontein** [comment]
@TSKushal thanks for your contribution!
@mraineri thanks for reviewing!

**Comment by TSKushal** [comment]
> @TSKushal thanks for your contribution! @mraineri thanks for reviewing!

Thank you 😄 @felixfontein @mraineri
